readthedocs/core/management/commands/contact_owners.py

```diff
@@ -0,0 +1,109 @@
+import logging
+from pathlib import Path
+from pprint import pprint
+from textwrap import dedent
+
+import markdown
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from readthedocs.core.utils.contact import contact_users
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
```

Comment:
This should have a docstring showing how to use it. 

Is there a good way to send arbitrary notifications? Or is everything required to be set in a template?

Reply:
Wanted to put this in the --help option, but looks like it's just a one-line description. I have added some examples in the docstring.

> Is there a good way to send arbitrary notifications? Or is everything required to be set in a template?

Not sure what you mean, we can send notifications with

```
contact_owners --notification notification.md
```

`notication.md` will be parsed through markdown first.